sympy/physics/control/lti.py

```diff
@@ -2314,7 +2325,7 @@ def __new__(cls, sys1, sys2=None, sign=-1):
                 Both `sys1` and `sys2` should be using the
                 same complex variable."""))
 
-        return super().__new__(cls, sys1, sys2, _sympify(sign))
+        return super(SISOLinearTimeInvariant, cls).__new__(cls, sys1, sys2, _sympify(sign))
```

Comment:
I may still rethink about inheriting `Feedback` from `TransferFunction`.

Reply:
Yes it should be from `TransferFunction`. I've done the changes.